js/eventloop/eventloop.go

```diff
@@ -178,10 +178,12 @@ func (e *EventLoop) Start(firstCallback func() error) error {
 		for promise := range e.pendingPromiseRejections {
 			// TODO maybe throw the whole promise up and get make a better message outside of the event loop
```

Comment:
Can this TODO be deleted now?

Reply:
```suggestion
```

Yeah I don't think this will be possible either way :thinking: